src/scripts/viewer/2d/graphUI.js

```diff
@@ -111,4 +115,12 @@ module.exports = function(svgRoot) {
     info.model.textColor = textColor;
     info.ui.dataSource(info.model);
   }
+
+  function getNodeRadius(node) {
+    var radius = defaultRadius;
+    if (showSize && node.data && node.data.dist && node.data.dist.unpackedSize) {
```

Comment:
Just curious - where does `unpackedSize` come from? I tried to explore [a random package](https://registry.npmjs.cf/shell-quote) from the `registry.npmjs.cf` and I don't see it there

Reply:
@anvaka I think this was introduced in the last 4 months or so, so only packages published since then will have this information populated. I [asked the npm team](https://npm.community/t/npm-registry-dist-unpackedsize/864) if there are plans to backfill.

I originally planned to make a HTTP request to my tool, [Package Phobia](https://packagephobia.now.sh), which will work for any arbitrary package, but I realized that I would need to submit a PR to the other repo (which would take more time and I'm lazy 😄)

I fallback to the 5px radius if the size was not found (this was the previous size used) .